packages/x/xfce4-taskmanager/package.yml

```diff
@@ -0,0 +1,22 @@
+name       : xfce4-taskmanager
+version    : 1.5.6
+release    : 1
+source     :
+    - https://archive.xfce.org/src/apps/xfce4-taskmanager/1.5/xfce4-taskmanager-1.5.6.tar.bz2 : 20979000761a41faed4f7f63f27bd18bb36fb27db4f7ecc8784a460701fb4abb
+homepage   : https://docs.xfce.org/apps/xfce4-taskmanager/start
+license    : GPL-2.0-or-later
+component  : desktop.xfce
+summary    : Easy to use task manager
+description: |
+    The xfce4-taskmanager component is an easy to use graphical task manager.
+builddeps  :
+    - pkgconfig(gtk+-3.0)
+    - pkgconfig(libxfce4ui-2)
+    - pkgconfig(libwnck-3.0)
+    - pkgconfig(xmu)
+setup      : |
+    %configure --sysconfdir=/usr/share
```

Comment:
Is overriding sysconfdir necessary?

Reply:
I'm thinking not. It doesn't provide any config files, so it probably doesn't do anything.